pkg/transforms/genericResourceConfig.go

```diff
@@ -9,17 +9,25 @@ type ExtractProperty struct {
 	metadataOnly bool
 }
 
+type ExtractEdge struct {
+	Name   string
+	ToKind string
+	Type   string
+}
```

Comment:
Use the EdgeType defined here. https://github.com/stolostron/search-collector/blob/e9c086cd1a57fc9bebba13caf740f142abea7044/pkg/transforms/transformer.go#L122-L132

```suggestion
type ExtractEdge struct {
	Name     string   // `json:"name,omitempty"`
	ToKind string // `json:"kind,omitempty"`
	Type   Type EdgeType // `json:"type,omitempty"`
}
```

Reply:
It's `toKind`
Bad copy/paste